src/docs/quick-start/test-wasm-wrappers/end-to-end-test.md

```diff
@@ -93,12 +71,7 @@ $snippet: js-e2e-test-init
 ## **Building and deploying a Wasm wrapper for testing with @polywrap/test-env-js**
 
 To invoke a Wasm wrapper in a test, we first need to build and deploy it. We can do this with `@polywrap/test-env-js` using
-the `buildAndDeployWrapper` function. The `buildAndDeployWrapper` function requires an absolute path to the directory of 
-your wrapper project that contains the `polywrap.yaml` manifest file, the values that were returned by `initTestEnvironment`, 
-and optionally an ENS domain name.
-
-The `buildAndDeployWrapper` function returns an ENS domain name and an IPFS CID. Either of these outputs can be used to call
-your wrapper. If an ENS domain name was not provided when calling `buildAndDeployWrapper`, a name will be randomly selected for you.
+the `buildAndDeployWrapper` function. 
```

Comment:
What about explaining the two ways of doing this, being the first one only building the wrapper and accessing it with file system URI and the other one deploying it and accessing it using ENS or IPFS URI

Reply:
I also noticed that you talked more about this below, I would say that here probably we can suggest to use the `buildWrapper` and then we can talk about the `buildAndDeploy`?